src/main/java/ui/GUIController.java

```diff
@@ -102,11 +117,43 @@ public void panelFilterExpressionChanged(FilterPanel changedPanel) {
                 .collect(Collectors.toList());
     }
 
-    private void updateAPIBox(UpdateRateLimitsEvent e) {
-        Platform.runLater(() -> apiBox.setText(String.format("%s/%s",
-                                                             e.remainingRequests,
-                                                             Utility.minutesFromNow(e.nextRefreshInMillisecs)))
-        );
+    private void updateRemainingRateEvent(UpdateRateLimitsEvent e) {
```

Comment:
this method is for updating of the remaining request and the next api quota renewal. Currently, it updates only the apibox. Updating of refreshRate is done using another method(updateSyncRefreshRate). The reason for their separation is because calculation of sync refresh rate should be only called once every refresh cycle. But updating of the apiBox could be called more than once.